src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,28 @@
+package duke.task;
+
+public class Event extends Task {
+    String eventTime;
```

Comment:
Good job in ensuring that code is kept [simple](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#practice-kissing) here.